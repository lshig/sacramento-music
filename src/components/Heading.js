import React from 'react';

export default function Heading() {
  return (
    <div className="heading">
      <div className="heading--wrapper">
        <h1>Sacramento, CA</h1>
        <h1>Meets</h1>
        <h1>Music and Code</h1>
      </div>
    </div>
  );
}
