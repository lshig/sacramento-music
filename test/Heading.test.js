import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import Heading from '../src/components/Heading';

describe('Heading', () => {
  describe('basic', () => {
    const wrapper = shallow(<Heading />);

    test('renders', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
