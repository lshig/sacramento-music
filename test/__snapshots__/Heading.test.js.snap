// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading basic renders 1`] = `
<div
  className="heading"
>
  <div
    className="heading--wrapper"
  >
    <h1>
      Sacramento, CA
    </h1>
    <h1>
      Meets
    </h1>
    <h1>
      Music and Code
    </h1>
  </div>
</div>
`;
